recipes/karawun/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "karawun" %}
+{% set version = "0.2.5.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/karawun-{{ version }}.tar.gz
+  sha256: 2e12897b4314e695c908e406ad954a3c3f0417769ac4c627a6827a77c59e28c0
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - importTractography = karawun.commandline:import_tractography_cl
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<=3.9
+  run:
+    - numpy >=1.13.0
+    - pydicom ==1.4.2
+    - python >=3.6,<=3.9
+    - simpleitk >=1.2.0,<=2.0.2
+
+test:
+  imports:
+    - karawun
+  commands:
+    - pip check
+    - importTractography --help
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/karawun/
+  summary: DICOM image, segmgmentation image and fibre object converter
+  license: AFL-2.0
+  license_file: LICENSE
```

Since it was tricky to find, would be good to add:
```suggestion
  license_file: LICENSE
  dev_url: https://github.com/DevelopmentalImagingMCRI/karawun
```